### PR TITLE
Does stuff that nuklear wants with notes and all that

### PR DESCRIPTION
**Pull request by ghost**
I've no clue if this'll work as local testing doesn't have it working, BUT I don't have a database so iunno, it should THEORETICALLY work though there's no reason why not - push to test server maybe?
- players that are new (I assume any player that has no notes is new) has join date, related CIDs & IPs logged in their notes

### CODE REVIEW

**Comment by Kurfursten** [comment]
Needs changelog

**Comment by ghost** [comment]
How in hell's name does this need a changelog

**Comment by Kurfursten** [comment]
Because your PR name and description don't say what it does

**Comment by ghost** [comment]
Maybe read the code then it's only 22 lines, it's not hard

**Comment by Kurfursten** [comment]
It's a matter of principle for the repo. Not everyone here CAN read the code, but anyone who plays on the server should be able to give input on PRs. I'm not singling you out: I've harassed bauds for failing to name/describe PRs in the past.

Don't you think you're acting childish by wasting time replying instead of charging the name/description, intentionally to obfuscate?

**Comment by ghost** [comment]
there you are

**Comment by Kurfursten** [comment]
Thank you

**Comment by clusterfack** [comment]
Okay, just no

**Comment by ghost** [comment]
rude

**Comment by clusterfack** [comment]
Let me see if I can think of a better way to do this

**Comment by ghost** [comment]
Fine with me

File: code/WorkInProgress/Cael_Aislinn/Rust/fuel_injector.dm [hunk 1]
```diff
@@ -256,7 +256,7 @@
 	set name = "Rotate Generator (Clockwise)"
 	set src in view(1)
 
-	if (usr.stat || usr.restrained()  || anchored || (usr.status_flags & FAKEDEATH))
+	if (usr.isUnconscious() || usr.restrained()  || anchored)
 		return
 
 	src.dir = turn(src.dir, -90)
```
Comment (N3X15) [inline code/WorkInProgress/Cael_Aislinn/Rust/fuel_injector.dm:262]: Okay, no, this is not an atomic commit.  

http://ss13.pomf.se/wiki/index.php/Getting_Your_Pull_Accepted#5._Pull_Requests_Must_Be_Atomic

You have unrelated bugfixes and a feature addition crammed into one PR.  We are not congress. You do not add "riders".

File: code/_onclick/ventcrawl.dm [hunk 2]
```diff
@@ -12,7 +12,7 @@ var/list/ventcrawl_machinery = list(/obj/machinery/atmospherics/unary/vent_pump,
 
 // Vent crawling whitelisted items, whoo
 /mob/living
-	var/canEnterVentWith = "/obj/item/weapon/implant=0&/obj/item/clothing/mask/facehugger=0&/obj/item/device/radio/borg=0&/obj/machinery/camera=0"
+	var/canEnterVentWith = "/obj/item/weapon/implant=0&/obj/item/clothing/mask/facehugger=0&/obj/item/device/radio/borg=0&/obj/machinery/camera=0&/mob/living/simple_animal/borer=0"
```
Comment (N3X15) [inline code/_onclick/ventcrawl.dm:15]: Another bugfix unrelated to the last

File: code/WorkInProgress/Cael_Aislinn/Rust/fuel_injector.dm (hunk 1 above)
Comment (ghost) [inline code/WorkInProgress/Cael_Aislinn/Rust/fuel_injector.dm:262]: I DIDN'T ADD THIS

File: code/_onclick/ventcrawl.dm (hunk 2 above)
Comment (ghost) [inline code/_onclick/ventcrawl.dm:15]: I DIDN'T ADD THIS

**Comment by ghost** [comment]
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA

**Comment by ihadtoregisterforthis** [comment]
> PJB3005 and others added some commits 11 days ago

**Comment by unid15** [comment]
try **git pull --rebase upstream Bleeding-Edge** @DrCelt

**Pull request closed without merging**